extensions/interactions/rules.spec.ts

```diff
@@ -50,13 +52,16 @@ describe('Rule spec services', function() {
     $provide.value('GraphUtilsService', new GraphUtilsService());
     $provide.value('FractionObjectFactory', new FractionObjectFactory());
     $provide.value('SetInputRulesService', new SetInputRulesService());
-    $provide.value('DragAndDropSortInputRulesService',
+    $provide.value(
```

Comment:
We don't have a style guide for `typescript` yet. Do we prefer to break down at first bracket here too like python?

Reply:
Yes please. Thanks!

Reply:
Noted, thanks!